cypress/components/confirm/confirm.cy.tsx

```diff
@@ -323,16 +341,10 @@ context("Testing Confirm component", () => {
       );
     });
 
-    it("should focus the dialog container", () => {
```

Comment:
Has this test been removed?

Reply:
Yes, there is no such blockFocusElement prop in confirm component now